programs/marginfi/src/instructions/marginfi_group/handle_bankruptcy.rs

```diff
@@ -55,7 +55,7 @@ pub fn lending_pool_handle_bankruptcy<'info>(
 
     if !bank.get_flag(PERMISSIONLESS_BAD_DEBT_SETTLEMENT_FLAG) {
         check!(
-            ctx.accounts.signer.key() == marginfi_group_loader.load()?.admin,
+            ctx.accounts.signer.key() == marginfi_group_loader.load()?.risk_admin,
```

Comment:
nit: either risk_admin OR regular admin probably fine here, but I don't think we're going to use the latter so nbd.